etl/transformer/transformer.go

```diff
@@ -26,26 +26,23 @@ import (
 	"github.com/kelindar/binary"
 )
 
-const (
-	// delta between pulses
-	pulseDelta uint16 = 10
-)
-
 // Transform transforms thr row JetDrops to canonical JetDrops
 func Transform(ctx context.Context, jd *types.PlatformJetDrops) ([]*types.JetDrop, error) {
-	// if no records per pulse
-	if len(jd.Records) == 0 {
-		return make([]*types.JetDrop, 0), nil
-	}
+	pulseData := getPulseData(jd.Pulse)
 
-	pulseData, err := getPulseData(jd.Records[0])
+	m, err := getRecords(jd)
 	if err != nil {
-		return nil, errors.Wrapf(err, "cannot get pulse data from record")
+		return nil, err
 	}
 
-	m, err := getRecords(jd.Records)
-	if err != nil {
-		return nil, err
+	log := belogger.FromContext(ctx)
+	for _, jetid := range jd.Pulse.Jets {
+		if _, ok := m[jetid]; ok {
```

Comment:
it will produce a lot of logs. I suggest to log error only if NOT ok. what do you think?

Reply:
yes. it will produce lots of **debug**